src/main/kotlin/day1/InputSuggestion.kt

```diff
@@ -0,0 +1,2254 @@
+5534
+2382
+6496
+
+4068
+5877
+4364
+4376
+1497
+1206
+1415
+2254
+1396
+2077
+2786
+5512
+2506
+
+3142
+1754
+5661
+2889
+3229
+6005
+2407
+4628
+3582
+1725
+5785
+4232
+5352
+1226
+5609
+
+2732
+9414
+9668
+4301
+3055
+8137
+8914
+3834
+9377
+
+14671
+10645
+
+4316
+4385
+4748
+7881
+7936
+7617
+5781
+7013
+1473
+1984
+5518
+
+4509
+5369
+6743
+7085
+5395
+9806
+5737
+1121
+
+7157
+5239
+7271
+1099
+8175
+10048
+2841
+8423
+
+27025
+36559
+
+3301
+5481
+6352
+3563
+2449
+3479
+1487
+3984
+1071
+1656
+6638
+3993
+1412
+
+3650
+4243
+4227
+5519
+3336
+2900
+3938
+6037
+3246
+5699
+3435
+5860
+1066
+4802
+
+1525
+1785
+6997
+4778
+2069
+3372
+6096
+6800
+5779
+4014
+3451
+4746
+
+3798
+2519
+6205
+3353
+7820
+6653
+7922
+4432
+1631
+6154
+1783
+
+1187
+5776
+2100
+7066
+4766
+3423
+1785
+5380
+6610
+2248
+4508
+4592
+
+2059
+2845
+12841
+3950
+9358
+
+9019
+2663
+7208
+7431
+4563
+3377
+7396
+2848
+2647
+
+16178
+6674
+13183
+11092
+15031
+
+4224
+5839
+6419
+6824
+4518
+4393
+3771
+3399
+1090
+6763
+7110
+4392
+
+1008
+5634
+4685
+5583
+5873
+1943
+3576
+4153
+3125
+4002
+4439
+1158
+4053
+3963
+4198
+
+3088
+1490
+7239
+5837
+1930
+1432
+1495
+5116
+7873
+3216
+7489
+
+8637
+14951
+19005
+14421
+
+4945
+7873
+8669
+7228
+4293
+6095
+6660
+6220
+1335
+
+4752
+7902
+3357
+1906
+3132
+7868
+7299
+4289
+7528
+7604
+7371
+
+4693
+2914
+13707
+9950
+4991
+10810
+
+30732
+
+8395
+4046
+8031
+2813
+1483
+8489
+4838
+6505
+4754
+5386
+
+10511
+1906
+8756
+1751
+4072
+
+18628
+14513
+13063
+19406
+
+1111
+4186
+4235
+1599
+1731
+1852
+3244
+2635
+5610
+3102
+5894
+5849
+5718
+5079
+4573
+
+51008
+
+25315
+17714
+
+1505
+8676
+2966
+4331
+5162
+8018
+4868
+4344
+4985
+
+11237
+8498
+6919
+9222
+2059
+9324
+2329
+
+5018
+8853
+7558
+4520
+4177
+3610
+2257
+1513
+5298
+
+6291
+1410
+2607
+3667
+7469
+1608
+7333
+2908
+6850
+2743
+
+6866
+5110
+8069
+7634
+6310
+4908
+5842
+5581
+4076
+1196
+6579
+
+1319
+3115
+2309
+3872
+5552
+1470
+3074
+3253
+3738
+6084
+3459
+4221
+5268
+6016
+
+7919
+1882
+11433
+18773
+
+1596
+3958
+3249
+2869
+1171
+1443
+2556
+3904
+5368
+6078
+5025
+4812
+4167
+2152
+4889
+
+7159
+1099
+6275
+3461
+3997
+4417
+8011
+1608
+7009
+1372
+6178
+
+2353
+12931
+7678
+6294
+16167
+
+10678
+8372
+5916
+11238
+11574
+13452
+
+2331
+2546
+5424
+4557
+1353
+3844
+1793
+3914
+3917
+4436
+4985
+3390
+3180
+5529
+1561
+
+6276
+1284
+1223
+1143
+4482
+3777
+6018
+6018
+5190
+5435
+1859
+6101
+4215
+1716
+
+5422
+9086
+8574
+6479
+6945
+10267
+6679
+5997
+
+2101
+7080
+3423
+6433
+3374
+10072
+5362
+5286
+
+3365
+11743
+9335
+7382
+10030
+10493
+11691
+
+3066
+2089
+2824
+5420
+2195
+2658
+3775
+2255
+5151
+3013
+5958
+5233
+6137
+2363
+
+10826
+1871
+15674
+13753
+
+7349
+11368
+10322
+17805
+
+2054
+1940
+6198
+4186
+2003
+6492
+5277
+6729
+7513
+4743
+7883
+
+45579
+
+1218
+9783
+6495
+7176
+9980
+11228
+
+27882
+
+3541
+7048
+6061
+6275
+3314
+1351
+1270
+7357
+5754
+3537
+2385
+3552"""
+val list = input.split("\n\n").map {
+    it.split("\n")
```

Comment:
suggested by Sebastian (@sebi_io) [here](https://www.youtube.com/watch?v=ntbsbqLCKDs)
```suggestion
    it.lines()
```